tests/integration_tests/validation/conformance_suite_configurations/efm_current.py

```diff
@@ -414,7 +414,6 @@
         '624-rendering/14-cash-flows/gd/14-cash-flows-gd-testcase.xml',
         '624-rendering/15-equity-changes/gw/15-equity-changes-gw-testcase.xml',
         '624-rendering/18-numeric/gd/18-numeric-gd-testcase.xml',
-        '626-rendering-syntax/626-03-no-matching-durations/626-03-no-matching-durations-testcase.xml',
```

Comment:
The change to the basepath handling seems to have to corrected it.  The testcase was not empty and contained a variation that appears to have been validating with errors for some time. Master run on conformance suite: 
https://github.com/Arelle/Arelle/actions/runs/8178762192/job/22363419074

Reply:
The testcase itself is actually not empty. It must have had a variation added in one of the updates since this config value was added last year.

It doesn't appear that the "testcase was expected to be empty but was not" scenario is covered by the current logic, so CI has been passing regardless.

As far as why this PR is now causing it to fail, it looks like there's an issue with the `get_document_id` changes. Once those are fixed we can leave EFM untouched for this PR and I'll PR a fix for the unexpected non-empty testcase scenario.